api/integrations/github/github.py

```diff
@@ -25,6 +26,21 @@
 logger = logging.getLogger(__name__)
 
 
+def handle_installation_deleted(payload: dict[str, typing.Any]) -> None:
```

Comment:
nit: `Any` has already been imported from `typing` so this should be like the below
```suggestion
def handle_installation_deleted(payload: dict[str, Any]) -> None:
```

Reply:
Done